tests/scipy_stats_test.py

```diff
@@ -176,6 +177,7 @@ def args_maker():
 
     self._CheckAgainstNumpy(scipy_fun, lax_fun, args_maker, check_dtypes=True)
     self._CompileAndCheck(lax_fun, args_maker, check_dtypes=True)
+    assert lax_fun(*args_maker()).shape == lax.broadcast_shapes(*shapes)
```

Comment:
I think you can revert the change to the test. The `_CheckAgainstNumpy` should be catching the mismatched shape but is not. I'm going to fix that part separately.

Reply:
Sure, thank you!